src/components/StrategicWords.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import data from './data.json';
+
```

Comment:
what's going on here? break this up into parts that make it easy for me to read.

Also, try to re-write this without using `let`. Don't use `let` or `var` until you're experienced enough to explain why and when you should.

Reply:
I would recommend looking at arr.reduce, which might be helpful here (as well as splitting this out into more bite-sized + well-named functions).